dojo/utils.py

```diff
@@ -2429,7 +2429,7 @@ def get_password_requirements_string():
     if bool(get_system_setting('number_character_required')):
         s += ', one number (0-9)'
     if bool(get_system_setting('special_character_required')):
-        s += ', one special chacter (()[]{}|\`~!@#$%^&*_-+=;:\'\",<>./?)'  # noqa W605
+        s += ', one special chacter (()[]{}|\\`~!@#$%^&*_-+=;:\'\",<>./?)'
```

Comment:
While you're at it... 😇 
```suggestion
        s += ', one special character (()[]{}|\\`~!@#$%^&*_-+=;:\'\",<>./?)'
```